asn1/InterledgerPacket.asn

```diff
@@ -9,6 +9,7 @@ IMPORTS
     FROM GenericTypes
 
     InterledgerProtocolPaymentData,
+    BestEffortPaymentData,
```

Comment:
Suggest this is called:

`InterledgerProtocolData `

Reply:
If we deprecate packet types 1-7, then it would still make sense to have type 8 be called 'error' and have type 9 be called something like 'request', and not just 'protocol'

Reply:
I think it should be called something like `InterledgerProtocolV2Data`. I don't think it's really a request/response protocol (though I might feel differently about that if we end up adding fulfillment data)

Reply:
:+1: for `InterledgerProtocolRequestData` for symmetry with `InterledgerProtocolErrorData`

👎 for adding versioning, that already implicit in the packet type numbering. (Unless a new packet is an explicit replacement for an old one, in which case I think it's required). This packet, by design, could carry more than payments.